tools/FakeItEasy.Build/FakeItEasy.Build.csproj

```diff
@@ -18,10 +18,11 @@
   <Target Name="GenerateToolPaths" BeforeTargets="BeforeBuild">
     <PropertyGroup>
       <ClassName>ToolPaths</ClassName>
-      <OutputFile>$(IntermediateOutputPath)\$(ClassName).cs</OutputFile>
+      <OutputFile>$(ClassName).cs</OutputFile>
     </PropertyGroup>
     <ItemGroup>
       <Tool Include="@(PackageReference)" Condition="'%(PackageReference.ToolName)' != ''" />
+      <Compile Remove="$(OutputFile)" />

```

Comment:
I have to remove the file before including it, because if it already existed, it would already be implicitly included. There might be a way to use a `Condition` for this (i.e. "include if not already included"), but it was easier this way.